server/src/api/community.rs

```diff
@@ -543,12 +649,29 @@ impl Perform<AddModToCommunityResponse> for Oper<AddModToCommunity> {
 
     let moderators = CommunityModeratorView::for_community(&conn, data.community_id)?;
 
-    Ok(AddModToCommunityResponse { moderators })
+    let res = AddModToCommunityResponse { moderators };
```

Comment:
is there a reason to construct this here instead of just doing `Ok(AddModToCommunityResponse { moderators})` at the end?

Reply:
oh nvm, i see the clone there